cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -81,7 +81,7 @@ func CreatePool(cStorPool *apis.CStorPool) error {
 func createPoolBuilder(cStorPool *apis.CStorPool) []string {
 	// populate pool creation attributes.
 	var createAttr []string
-	createAttr = append(createAttr, "create")
+	createAttr = append(createAttr, "create", "-f")
```

Comment:
What is the need to have the force create flag? Need to be sure about the reason for doing so. For now, add a comment specifying the reason for doing so.

Reply:
will add